### ISSUE TITLE
FIRMWARE_VERSION is missing

### ISSUE THREAD

**Issue by victorevector** [Robo3D/Marlin#3]
an M115 command does not produce a `FIRMWARE_VERSION` value. Octoprint-FirmwareUpdater needs that key in order for it to perform a version check.

This is the mainboard's response to the M115 command:

~~~
FIRMWARE_NAME:Marlin 1.1.0-RC8 (Github) SOURCE_CODE_URL:https://github.com/Robo3D/Marlin-C2 PROTOCOL_VERSION:1.0 MACHINE_TYPE:RoboC2 EXTRUDER_COUNT:1 UUID:cede2a2f-41a2-4748-9b12-c55c62f367ff
~~~

**Comment by AllenMcAfee** [comment]
fixed in 1.1.4

added FIRMWARE_VERSION to M115 output

**Comment by AllenMcAfee** [comment]
Adding PROTOCOL_VERSION to differentiate between R2 and C2 builds.
